### PR TITLE
blacklist regex env var instead of list of urls in file

### PR DESCRIPTION
**Pull request by seppo0010**

### CODE REVIEW

File: builder/main.go [hunk 1]
```diff
@@ -90,14 +56,27 @@ func main() {
 		"actuaciones": len(args.exp.Actuaciones),
 	}).Printf("finished")
 
+	blacklist := os.Getenv("BLACKLIST_REGEX")
```
Comment (xaiki) [inline builder/main.go:59]: This mixes env vars and args to pass arguments.
We should choose one and stick towit.

**Review by xaiki — CHANGES_REQUESTED** [review]
It'd be great to pr against gh-actions branch so we can make sure it actually does what we want.

**Comment by seppo0010** [comment]
Warning! The base is gh-actions and not main.

File: builder/extracttext/lib.go [hunk 2]
```diff
@@ -84,7 +84,7 @@ func getDocumentPlainText(p string) (string, error) {
 }
 
 func pdftohtml(p string) error {
-	cmd := exec.Command("pdftohtml", p)
+	cmd := exec.Command("pdftohtml", "-c", p)
```
Comment (xaiki) [inline builder/extracttext/lib.go:87]: looks unrelated

**Review by xaiki — CHANGES_REQUESTED** [review]
this is missing the actions part.

**Comment by xaiki** [comment]
commit history looks like it needs a cleanup.

File: builder/extracttext/lib.go (hunk 2 above)
Comment (seppo0010) [reply builder/extracttext/lib.go:87]: you noticed

**Comment by seppo0010** [comment]
What do you mean? https://github.com/odia/juscaba/pull/36#pullrequestreview-804779024

**Comment by xaiki** [comment]
i saw many commits, dunno why.

**Pull request merged**